.devops/nodo-db-migration-pipelines.yml

```diff
@@ -0,0 +1,234 @@
+pr: none
+trigger: none
+#trigger:
+#  - develop
+
+pool:
+  vmImage: 'ubuntu-latest'
+
+parameters:
+  - name: k8sEnv
+    displayName: K8s Environment
+    type: string
+    default: dev
+    values:
+      - dev
+      - uat
+      - prd
+  - name: dbVersion
+    displayName: Database Version
+    type: string
+  - name: lbContexts
+    displayName: Liquibase Contexts
+    type: string
+    default: '_'
+  - name: lbLogLevel
+    displayName: Liquibase Log Level
+    type: string
+    default: INFO
+    values:
+      - INFO
+      - FINE
+      - WARNING
+      - SEVERE
+
+variables:
+  ${{ if eq(parameters.k8sEnv, 'dev') }}:
+    AZURE_SERVICE_CONNECTION: '$(TF_DEV_AZURE_SERVICE_CONNECTION)'
+    JDBC_URL: jdbc:postgresql://pagopa-d-weu-nodo-flexible-postgresql.postgres.database.azure.com:5432/nodo?sslmode=require
+    KEY_VAULT_NAME: pagopa-d-nodo-kv
+  ${{ elseif eq(parameters.k8sEnv, 'uat') }}:
+    AZURE_SERVICE_CONNECTION: '$(TF_UAT_AZURE_SERVICE_CONNECTION)'
+    JDBC_URL: jdbc:postgresql://pagopa-u-weu-nodo-flexible-postgresql.postgres.database.azure.com:5432/nodo?sslmode=require
+    KEY_VAULT_NAME: pagopa-u-nodo-kv
+  ${{ elseif eq(parameters.k8sEnv, 'prd') }}:
+    AZURE_SERVICE_CONNECTION: '$(TF_PROD_AZURE_SERVICE_CONNECTION)'
+    JDBC_URL: jdbc:postgresql://pagopa-p-weu-nodo-flexible-postgresql.postgres.database.azure.com:5432/nodo?sslmode=require
+    KEY_VAULT_NAME: pagopa-p-nodo-kv
+  CFG_USERNAME: cfg
+  CFG_SCHEMA: cfg
+  OFFLINE_USERNAME: offline
+  OFFLINE_SCHEMA: offline
+  ONLINE_USERNAME: online
+  ONLINE_SCHEMA: online
+  RE_USERNAME: re
+  RE_SCHEMA: re
+  WFESP_USERNAME: wfesp
+  WFESP_SCHEMA: wfesp
+
+stages:
+  - stage: prd_approval
+    pool: server
+    displayName: "Manual approval for prod"
+    condition: eq('${{ parameters.k8sEnv }}', 'prd')
+    jobs:
+      - job: prd_approval
+        pool: server
+        condition: eq('${{ parameters.k8sEnv }}', 'prd')
+        displayName: "Approval"
+        timeoutInMinutes: 6
+        steps:
+          - task: ManualValidation@0
+            timeoutInMinutes: 5
+            inputs:
+              notifyUsers: |
+                test@test.com
```

Comment:
change with slack and google internal grp

Reply:
done